config/locales/es.yml

```diff
@@ -245,6 +245,7 @@ es:
   edit_moment: "Editar Momento"
   tagged: "Etiquetar Momento"
   new: "Nuevo Momento"
+  secret_share: 'Compártelo en Secreto'
```

Comment:
Looks like we're missing secret_share.create and secret_share.cancel. Ahh this is why your Circle build is failing.

Reply:
Yes, we added secret_share keys that need to be filled in for each translation. Shall we call on the translators to address this and fill in the missing keys?

Reply:
I put some translated missing keys as place holders.